src/data/shortlist.h

```diff
@@ -257,6 +257,13 @@ class LexicalShortlistGenerator : public ShortlistGenerator {
       for(auto& it : data_[i])
         indexSet.insert(it.first);
     }
+    // Ensure that the generated vocabulary items from a shortlist are a multiple-of-eight
```

Comment:
Could you also add why this is required/what does require this?

Reply:
Changed